### PR TITLE
Modifying default log level to ERROR

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
bugzilla: 1654183
tendrl-bug-id: Tendrl/node-agent#860

Signed-off-by: GowthamShanmugasundaram <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@shtripat @nthomas-redhat please review

File: etc/tendrl/node-agent/node-agent-dev.conf.yaml [hunk 1]
```diff
@@ -2,7 +2,6 @@
 etcd_port: 2379
 etcd_connection: 0.0.0.0
 log_cfg_path: /etc/tendrl/node-agent/node-agent_logging.yaml
-log_level: DEBUG
 package_source_type: pip
```
Comment (shtripat) [inline etc/tendrl/node-agent/node-agent-dev.conf.yaml]: why remove the above line? In dev env DEBUG is fine.

File: etc/tendrl/node-agent/node-agent.conf.yaml.sample [hunk 2]
```diff
@@ -9,7 +9,6 @@ etcd_connection: 0.0.0.0
 
 
 log_cfg_path: /etc/tendrl/node-agent/node-agent_logging.yaml
-log_level: DEBUG
 package_source_type: rpm
```
Comment (shtripat) [inline etc/tendrl/node-agent/node-agent.conf.yaml.sample]: Why remove here?

File: etc/tendrl/node-agent/node-agent-dev.conf.yaml (hunk 1 above)
Comment (GowthamShanmugam) [reply etc/tendrl/node-agent/node-agent-dev.conf.yaml]: No this is not used in anywhere, the log is completely  depending on sys log file

File: etc/tendrl/node-agent/node-agent.conf.yaml.sample (hunk 2 above)
Comment (GowthamShanmugam) [reply etc/tendrl/node-agent/node-agent.conf.yaml.sample]: log_level is not used anywhere,  all logging is configured using only logging.yaml.syslog.sample

File: etc/tendrl/node-agent/node-agent-dev.conf.yaml (hunk 1 above)
Comment (shtripat) [reply etc/tendrl/node-agent/node-agent-dev.conf.yaml]: Ack

File: etc/tendrl/node-agent/node-agent.conf.yaml.sample (hunk 2 above)
Comment (shtripat) [reply etc/tendrl/node-agent/node-agent.conf.yaml.sample]: Ack

**Comment by shtripat** [comment]
@GowthamShanmugam fix the pep8 issues

**Pull request merged**